dashboard/app/views/levels/editors/_netsim.html.haml

```diff
@@ -22,37 +26,102 @@
     %tr
       %td= boolean_check_box f, :show_add_router_button
       %td= f.label :show_add_router_button, 'Show "Add Router/Room" Button'
-      %td Whether the "Add Router" (or "Add Room") button should be available to students.
+      %td
+        Whether the "Add Router" (or "Add Room") button should be available
+        to students.
 
 %h3 Simulation Configuration
 .field
   = f.label :message_granularity, 'Message Granularity'
-  %p Whether the simulator puts a single bit into storage at a time, or a whole packet. Should use 'bits' for variant 1 (levels about the coordination problem), and 'packets' for levels where the coordination problem is abstracted away.
+  %p
+    Whether the simulator puts a single bit into storage at a time, or a whole
+    packet. Should use 'bits' for variant 1 (levels about the coordination
+    problem), and 'packets' for levels where the coordination problem is
+    abstracted away.
   = f.select :message_granularity, options_for_select(@level.class.message_granularity_options, @level.message_granularity)
+  %p
+    Note: Using "packet" granularity just means a whole message is sent when
+    you click Send.  It does not necessarily:
+    %ul
+      %li Surface the concept of a "packet" to the student
+      %li Limit packet size (see "Default packet size")
+      %li Let the student compose multiple packets (see "Show 'Add Packet' button")
+      %li Use headers or packet counts (see "Client packet header format")
```

Comment:
Additional description on the message_granularity field, because it's caused some confusion already.